templates/aem_type.erb

```diff
@@ -13,23 +13,34 @@
 . /etc/sysconfig/network
 
 # Check that networking is up.
-[ ${NETWORKING} = "no" ] && exit 0
+[ ${NETWORKING} = "no" ] && exit 1
 
 SCRIPTNAME=$0
 INSTANCENAME="<%= @instance_name %>"
-PROG=$"CQ5 ${INSTANCENAME} instance."
+PROG="AEM ${INSTANCENAME} instance."
 STARTUPDIR="<%= scope['::adobe_em6::params::dir_aem_install'] %>/${INSTANCENAME}/crx-quickstart"
-
-[ -f /etc/sysconfig/${INSTANCENAME} ] && . /etc/sysconfig/${INSTANCENAME}
+AEMUSER="<%= scope['::adobe_em6::params::aem_user'] %>"
+LOCK="/var/lock/subsys/${SCRIPTNAME}"
 
 start() {
-        echo -e $"Starting:\n ${PROG} "
-        su - <%= scope['::adobe_em6::params::aem_user'] %> -c "${STARTUPDIR}/bin/start"
+  [ -f "${LOCK}" ] && exit 0
+  touch "${LOCK}"
+  log "$(date) - Starting: ${PROG}"
+  su - ${AEMUSER} -c "${STARTUPDIR}/bin/start"
 }
 
 stop() {
-        echo -e $"Shutting down:\n ${PROG} "
-        su - <%= scope['::adobe_em6::params::aem_user'] %> -c "${STARTUPDIR}/bin/stop"
+  log "$(date) - Shutting down: ${PROG}"
+  su - ${AEMUSER} -c "${STARTUPDIR}/bin/stop"
+  wait
+  #Let things settle a bit
+  sleep 5
```

Comment:
There isn't a better way to know things are settled than to blindly sleep for 5 seconds?